lib/Infer/AbstractInterpreter.cpp

```diff
@@ -1114,6 +1114,10 @@ namespace souper {
         auto RB1 = RB.findRestrictedBits(I->Ops[1]);
         // Take bivalent bits of opposite operand to independent variables
         for (auto V : IVARS) {
+          if (V->Width != I->Ops[0]->Width ||
+              V->Width != I->Ops[1]->Width) {
```

Comment:
why is this happening?

Reply:
I didn't originally account for inputs with differing bitwidth.

Reply:
I just want to make sure you're not hiding initialization bugs

Reply:
Not here, there are a couple of similar bugs which I am fixing now which are problematic.